types/vfile-location/index.d.ts

```diff
@@ -1,12 +1,13 @@
 // Type definitions for vfile-location 2.0
 // Project: https://github.com/vfile/vfile-location
 // Definitions by: Ika <https://github.com/ikatyang>
+//                 Junyoung Choi <https://github.com/rokt33r>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.4
```

Comment:
Since this depends on vfile, its TypeScript version should also bump to 3.0

Reply:
Fixed!